### ISSUE TITLE
Request: disallow "/" as custom rewrite

### ISSUE THREAD

**Issue by dingman** [pods-framework/pods#5744]
I was adjusting some permalinks earlier today and I set my "Custom Rewrite Slug" to be "/" and suddenly all sorts of 404 errors on my site.

In short, I'm requesting that "Custom Rewrite Slug" disallow the usage of "/" by itself, as it has unknown consequences.

**Comment by JoryHogeveen** [comment]
Hi @dingman 
I think it stands for itself that `/` would not work.
Not sure whether it's worth to put a verification procedure for such values.

**Comment by JoryHogeveen** [comment]
@sc0ttkclark What do you think?

**Comment by dingman** [comment]
@sc0ttkclark had another case where this happened to someone else. What are your thoughts on this?

https://podswp.slack.com/archives/C02SVLHQF/p1608761447488200

**Comment by sc0ttkclark** [comment]
Fixed via 282441112c79693a245c1d67f5bd84e1ec41aa8c

Enforcing on pod save and forcing as blank if it sees "/"
